contrib/runners/mistral_v2/query/mistral_v2.py

```diff
@@ -105,10 +119,17 @@ def _get_workflow_tasks(self, exec_id):
         :type exec_id: ``str``
         :rtype: ``list``
         """
-        wf_tasks = [
-            self._client.tasks.get(task.id)
-            for task in self._client.tasks.list(workflow_execution_id=exec_id)
-        ]
+        try:
+            wf_tasks = [
```

Comment:
I would personally prefer to make this a regular for loop, because list comprehensions are usually only intended to be used for simple transformations and in cases like that when you have exception handling, etc. regular for loop is more clear and obvious.